extensions/vscode/src/utils/errorTypes.ts

```diff
@@ -162,6 +163,15 @@ export type ErrInvalidConfigFiles = MkErrorDataType<
 export const isErrInvalidConfigFile =
   mkErrorTypeGuard<ErrInvalidConfigFiles>("invalidConfigFile");
 
+// Invalid configuration file(s)
+export type ErrCredentialsReset = MkErrorDataType<"credentialCorruptedReset">;
+export const isErrCredentialsReset = mkErrorTypeGuard<ErrInvalidConfigFiles>(
+  "credentialCorruptedReset",
+);
+export const errCredentialsResetMessage = () => {
+  return "Stored credentials for Posit Publisher found in an unrecognizable state. A reset was required in order to proceed.";
+};
```

Comment:
The current wording might leave the user trying to figure out how to reset the credentials even though we have already done it for them. It's long and could use some further refinement, but how about something like this?
>Unable to use existing stored credentials for Posit Publisher (reason: unrecognizable state). The existing credentials have been deleted. Recreate them using the Credentials section or the Command Pallet's "Posit Publisher: New Credential" command.

Reply:
I agree it should be very clear that something has already been done. Another suggestion for the re-word:

> Unrecognizable credentials for Posit Publisher were found and removed. Credentials may need to be recreated.

I went with "removed" instead of "deleted" to be a bit softer but still clear that they are no longer on the system. I left the recreation message at the end since it will be collapsed most of the time, and since they have already made credentials (we know this because they needed a reset) did not include instructions.

Reply:
Thanks for the feedback, I agree. That's a much better warning message.